R/Prepare_Census_Tables.R

```diff
@@ -99,31 +186,24 @@ universeList$type[grepl("housing unit", universeList$stub, ignore.case = T)] <-
 
 ## Flag additional tables
 #allocation variables (https://www.census.gov/programs-surveys/acs/methodology/sample-size-and-data-quality/item-allocation-rates-definitions.html)
-tableList$flag[grepl("allocat", tableList$stub, ignore.case = T)] <- T
+variables$flag[grepl("allocat", variables$tableStub, ignore.case = T)] <- T
 
 #Response rate information
-tableList$flag[grepl("response rate", tableList$stub, ignore.case = T)] <- T
+variables$flag[grepl("response rate", variables$tableStub, ignore.case = T)] <- T
 
 #coverage rate information
-tableList$flag[grepl("coverage rate", tableList$stub, ignore.case = T)] <- T
+variables$flag[grepl("coverage rate", variables$tableStub, ignore.case = T)] <- T
 
 #unweighted sample sizes
-tableList$flag[grepl("^Unweighted", tableList$stub, ignore.case = T)] <- T
-
+variables$flag[grepl("^Unweighted", variables$tableStub, ignore.case = T)] <- T
 
 
 ## Filter out flagged tables from each dataset
-drop <- tableList$tableID[tableList$flag == T]
-
-tableList <- tableList[!tableList$tableID %in% drop,]
-variableList <- variableList[!variableList$tableID %in% drop,]
-universeList <- universeList[!universeList$tableID %in% drop,]
+variables <- variables[variables$flag == F,]
 
 
 
 ####  Save  ####
-saveRDS(tableList, file = "Data/Census_tables.rds")
-saveRDS(variableList, file = "Data/Census_variables.rds")
-saveRDS(universeList, file = "Data/Census_universes.rds")
+saveRDS(variables, file = "Data/Census_variables.rds")
```

Comment:
I think this is a huge improvement in workflow, in that we no longer need to import three separate `.rds` files when creating the Shiny app. Thanks for thinking of this, Isaac!

Reply:
For sure. In retrospect it's way more intuitive!